### PR TITLE
Issue #14715: Enforce new naming convention on methodtypeparametername in IT area

### PR DESCRIPTION
**Pull request by Kexin2000**
Part of #14715 
rename xpath it regression input files in MethodTypeParameterName module

### CODE REVIEW

File: src/it/resources/org/checkstyle/suppressionxpathfilter/methodtypeparametername/InputXpathMethodTypeParameterName1.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package org.checkstyle.suppressionxpathfilter.methodtypeparametername;
 
-public class SuppressionXpathRegressionMethodTypeParameterName1{
+public class InputXpathMethodTypeParameterName1{
```
Comment (MANISH-K-07) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/methodtypeparametername/InputXpathMethodTypeParameterName1.java:3]: Is it possible to find a better suffix than "1" ?
Please try to find a proper name for `InputXpathMethodTypeParameterName1` --> `InputXpathMethodTypeParameterNameXxxx`
Same for all others....

**Review by MANISH-K-07 — COMMENTED** [review]
@Kexin2000 , great !!
Please do us a favor and change PR title to 
`Issue #14715: Enforce new naming convention on methodtypeparametername in IT area`
just mentioning module name should be enough to make it look less clumsy ;)

File: src/it/resources/org/checkstyle/suppressionxpathfilter/methodtypeparametername/InputXpathMethodTypeParameterName1.java (hunk 1 above)
Comment (Kexin2000) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/methodtypeparametername/InputXpathMethodTypeParameterName1.java]: You make sense. done

**Review by romani — APPROVED** [review]
ok to merge

>test (java-ast-visitor)

restarted

**Review by rnveach — APPROVED** [review]

**Pull request merged**